.s2i/bin/assemble

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -e
+
+# The assemble script builds the application artifacts from a source and
+# places them into appropriate directories inside the image.
+
+# Execute the default S2I script
+/usr/libexec/s2i/assemble
+
+echo "---> Upgrade pip to the latest version ..."
+pip install --upgrade pip
+
+echo "---> Installing poetry ..."
+pip install poetry
+
+# Don't create virtual environments
+poetry config virtualenvs.create false
+
+echo "---> Installing datanommer subdirectories ..."
+for subpackage in $(find .  -mindepth 2 -type f -name pyproject.toml -exec dirname {} \;); do
+    pushd $subpackage
+    echo "Installing python package in $(pwd)"
+    poetry install
```

Comment:
Please add `--no-dev` because we don't need the development tools

Reply:
Ok